### PR TITLE
add other CI/CD platforms

### PR DESCRIPTION
**Pull request by Moshe-Immerman**

### CODE REVIEW

**Comment by akullpp** [comment]
If an entry has a similar scope as other entries in the same category, the description must state the unique features that distinguishes it from the other entries.

**Comment by Moshe-Immerman** [comment]
@akullpp 

* Buddy is pretty much the only hosted platform with on-premise option (Other open-source, on-premise options may provide managed hosting, but it isn't quite the same)
* Codefresh - is about building, testing and deploying docker  images (Docker-Native) Concourse, CircleCI, and Semaphore merely support docker
* Screwdriver.cd - will remove

File: README.md [hunk 1]
```diff
@@ -215,6 +215,9 @@ A curated list of awesome Java frameworks, libraries and software.
 *Tools which support continuously building, testing and releasing applications.*
 
 * [Bamboo ![c]](https://www.atlassian.com/software/bamboo) - Atlassian's solution with good integration of their other products. You can either apply for an open-source license or buy it.
+* [Buddy ![c]](https://buddy.works/) - Hosted CI/CD platform with on-premise option.
+* [CircleCI ![c]](https://circleci.com) - Hosted CI/CD platform with docker and OSX support. Concurrent builds rather than project based billing model.
```
Comment (jbduncan) [inline README.md]: @Moshe-Immerman By "OSX", do you mean "Mac OS X"? Where does it say on the website that CircleCI supports "OSX"? How does it do so? Why is this useful for Java developers? And if you did mean "Mac OS X", how about saying "macOS" (the new official name) instead?

Comment (Moshe-Immerman) [inline README.md]: @jbduncan https://circleci.com/mobile/osx/ - OSX is the term that they use

Comment (jbduncan) [inline README.md]: Okay, that's fine, but then we should split "OSX" into "OS X" as they do on the CircleCI website.

**Comment by akullpp** [comment]
All these resources are not really Java-centric. I think about deleting the whole section.

**Pull request closed without merging**